### ISSUE TITLE
Homepage link "Cowden Syndrome" is not active in dev_22.06 due to a change in EFO_ID

### ISSUE THREAD

**Issue by buniello** [opentargets/issues#2639]
While doing some QC on the dev version of platform 22-06 release, I have noticed that the [homepage link "cowden syndrome"](https://platform.dev.opentargets.xyz/disease/Orphanet_201/associations) does not work.

This is due to a change in its EFO_ID, which has now become [label: 'Cowden Disease', id: 'MONDO_0016063'](https://platform.dev.opentargets.xyz/disease/MONDO_0016063/associations) - from label: 'Cowden Syndrome', id: 'Orphanet_201'. 
NB: Orphanet_201 link is still active in 22.04.

**Comment by JarrodBaker** [comment]
The ontology widget for MONDO_0016063 is also broken.

**Comment by ireneisdoomed** [comment]
Neither does any evidence page for MONDO_0016063 https://platform.dev.opentargets.xyz/evidence/ENSG00000134086/MONDO_0016063

**Comment by d0choa** [comment]
Please update:
https://github.com/opentargets/ot-ui-apps/blob/05dd1f150718107a23c67bdf0d51ef61378fc14a/apps/platform/src/pages/HomePage/searchExamples.js

and not:
https://github.com/opentargets/ot-ui-apps/blob/05dd1f150718107a23c67bdf0d51ef61378fc14a/apps/platform/src/pages/HomePage/searchExamples.js

We will be archiving `platform-app` after this release. @DSuveges can you open a PR to FE once you have finished your review?

**Comment by DSuveges** [comment]
I'll open a PR with these updates. 

```
+-------------+---------------+
|           id|           name|
+-------------+---------------+
|MONDO_0018997|Noonan syndrome|
|MONDO_0016063| Cowden disease|
+-------------+---------------+
```

The targets looks fine.

**Comment by DSuveges** [comment]
Solved by [PR#11](https://github.com/opentargets/ot-ui-apps/pull/11)
